apps/Dataman/src/components/useLocation.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";

```

Comment:
this file does not need to be .tsx -> .ts is better

Reply:
Move to src/hooks/...

Reply:
done